build/common/commands/new.py

```diff
@@ -0,0 +1,67 @@
+import os, frappe, json
+
+from frappe.commands.site import _new_site
+
+site_name = os.environ.get("SITE_NAME", 'site1.localhost')
+mariadb_root_username = os.environ.get("DB_ROOT_USER", 'root')
+mariadb_root_password = os.environ.get("DB_ROOT_PASSWORD", 'admin')
+force = True if os.environ.get("FORCE", None) else False
+
+frappe.init(site_name, new_site=True)
+
+_new_site(
+    None,
+    site_name,
+    mariadb_root_username=mariadb_root_username,
+    mariadb_root_password=mariadb_root_password,
+    admin_password=os.environ.get("ADMIN_PASSWORD", 'admin'),
+    verbose=True,
+    install_apps=[],
```

Comment:
ERPNext is not initialised even when present in `apps.txt`.

Reply:
Also it would be nice if the new command was issued if no site is configured.

Reply:
> Also it would be nice if the new command was issued if no site is configured.

Will have to script it. Keeping things simple for first production release. Hope this gets merged first and images are built automatically for people to experiment. We'll improve as we (community users) use it.

Reply:
> ERPNext is not initialised even when present in `apps.txt`.

fixed it